docs/xxshownativestacksymbols.md

```diff
@@ -0,0 +1,44 @@
+<!--
+* Copyright (c) 2017, 2022 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:\[+|-\]ShowNativeStackSymbols
+
+This option controls whether Java&reg; dumps show the function names of the native stack symbols in native call stacks.
+
+## Syntax
+
+        -XX:-ShowNativeStackSymbols
+        -XX:+ShowNativeStackSymbols=<value>
+
+| Setting                    | Value  |Effect|Default|
+|----------------------------|--------|------|:-------:|
+|`-XX:-ShowNativeStackSymbols`|       |Don't show native stack symbols in response to a signal, such as a fault signal      |   |
```

Comment:
This isn't quite right. In `basic` mode, all symbols are shown for a faulting or signaled thread. With this option no native symbols are displayed, not even for a faulting or signaled thread; suggest:

> Don't show native stack symbols, not even in response to a signal, such as a fault signal